spec/factories/vacancies.rb

```diff
@@ -112,6 +112,16 @@
       expires_at { Faker::Time.backward(days: 6) }
     end
 
+    trait :expired_yesterday do
+      expires_on { Time.zone.yesterday.midday }
```

Comment:
should this be same range as expired_tomorrow i.e `expires_on { Time.zone.yesterday.end_of_day }` ?